login.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="reset.css">
+</head>
+<body>
+    <div class="login-box">
+        <a href="/index.html">
+            <img class="login-pandalogo" src="/image/logo.png" alt="pandalogo">
+        </a>
+        <form action="/login.html" method="POST">
```

Comment:
💊 제안
폼 태그의 action 속성은 form 데이터값을 보낼 URI로 아직 배우지 않은 API를 입력해주시는 속성입니다~
지금은 비워두시면됩니다~

[mdn form action](https://developer.mozilla.org/ko/docs/Web/HTML/Element/form#action)

```suggestion
        <form>
```